### PR TITLE
Non-doc page layout

### PR DESCRIPTION
**Pull request by Duslerke**
# What:
- Installed necessary styling packages
- Created Layout component.
- Created a header.
- Made header also be top nav bar.

image

# Why:
- It makes sense to extract all this into component given it's going to be used by more than a single page.

# Notes:
- While doing the side navbar, I realised that govuk width container is fundamentally incompatible with the docusaurus style I'm aiming to pull off. So likely, another layout component will be needed that's dedicated to doc pages.

### CODE REVIEW

**Comment by paralonas** [comment]
Yes, good, but Angular is better.

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "printWidth": 120
```
Comment (paralonas) [inline .prettierrc:2]: What is this?

**Review by paralonas — APPROVED** [review]
Yes, good. Angular is better.

File: .prettierrc (hunk 1 above)
Comment (Duslerke) [reply .prettierrc:2]: It's Prettier config. Made the code line width bigger so the formatting doesn't look as ugly as the default one.

**Pull request merged**